frontend/app/work_packages/directives/inplace_editor/custom/display/inplace-display-spent-time-directive.js

```diff
@@ -44,8 +44,9 @@ module.exports = function(EditableFieldsState) {
       };
     },
     controllerAs: 'customEditorController',
-    link: function(scope, element, attrs, displayPaneController) {
-      scope.displayPaneController = displayPaneController;
+    link: function(scope, element, attrs, controllers) {
+      scope.displayPaneController = controllers[0];
```

Comment:
@myabc I actually wrote something like (scope, element, attrs, [displayPaneController, fieldController]) then tried to understand what language I am hallucinating